tests/_setup/perf-fixtures/css-imports/entry.css

```
@@ -0,0 +1,14 @@
+/*
+ * Exercises every CSS @import variant the dependency-tree walker
+ * supports. Note: bare `@import url(./x.css)` (no quotes) is NOT
+ * followed — quote your url() arguments.
+ */
+@import "./double-quoted.css";
+@import './single-quoted.css';
+@import url("./url-double-quoted.css");
```

Stylelint (17.9.0)

[error] Expected "url("./url-double-quoted.css")" to be ""./url-double-quoted.css"" (import-notation)

(import-notation)